examples/README.md

```diff
@@ -0,0 +1,9 @@
+# Optimizely Go SDK Examples
+
+### Profiling
+Prerequisite: `go get github.com/pkg/profile`
+
+DATAFILES_DIR env var has to be set to point to the path where 100_entities.json is located
```

Comment:
Should probably add to the example code snippets below too.

Reply:
add what ?

Reply:
I added a quick description. i am sure there will be more, feel free to add more content if you like